src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+        Machine truck = new Truck();
+        Machine[] machines = {bulldozer, excavator, truck};
```

Comment:
I'd recommend creating bulldozer, excavator, and truck directly within array initializer so that avoid creating redundant variables